docs/guide/index.md

```diff
@@ -38,10 +38,11 @@ test('displays message', () => {
 
 Vue Test Utils is commonly used with a test runner. Popular test runners include:
 
-- [Vitest](https://vitest.dev/) 
-- [Cypress](https://cypress.io/) 
+- [Vitest](https://vitest.dev/). Terminal based.
```

Comment:
Vitest also has a browser mode (experimental)

Reply:
Oh neat, I did not know that.